easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,1475 @@
+    }),
+    ('Package::Stash::XS', '0.29', {
+        'source_tmpl': 'Package-Stash-XS-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/E/ET/ETHER'],
+        'checksums': ['d3676ba94641e03d6a30e951f09266c4c3ca3f5b58aa7b314a67f28e419878aa'],
+    }),
+    ('Set::Array', '0.30', {
+        'source_tmpl': 'Set-Array-%(version)s.tgz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RS/RSAVAGE'],
+        'checksums': ['d9f024c8e3637feccdebcf6479b6754b6c92f1209f567feaf0c23818af31ee3c'],
+    }),
+    ('boolean', '0.46', {
+        'source_tmpl': 'boolean-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/I/IN/INGY'],
+        'checksums': ['95c088085c3e83bf680fe6ce16d8264ec26310490f7d1680e416ea7a118f156a'],
+    }),
+    ('Number::Format', '1.75', {
+        'source_tmpl': 'Number-Format-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/W/WR/WRW'],
+        'checksums': ['82d659cb16461764fd44d11a9ce9e6a4f5e8767dc1069eb03467c6e55de257f3'],
+    }),
+    ('Data::Stag', '0.14', {
+        'source_tmpl': 'Data-Stag-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/C/CM/CMUNGALL'],
+        'checksums': ['4ab122508d2fb86d171a15f4006e5cf896d5facfa65219c0b243a89906258e59'],
+    }),
+    ('Test::NoWarnings', '1.04', {
+        'source_tmpl': 'Test-NoWarnings-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AD/ADAMK'],
+        'checksums': ['638a57658cb119af1fe5b15e73d47c2544dcfef84af0c6b1b2e97f08202b686c'],
+    }),
+    ('Crypt::DES', '2.07', {
+        'source_tmpl': 'Crypt-DES-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DP/DPARIS'],
+        'checksums': ['2db1ebb5837b4cb20051c0ee5b733b4453e3137df0a92306034c867621edd7e7'],
+    }),
+    ('Exporter', '5.73', {
+        'source_tmpl': 'Exporter-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['c5b6e96b0bda7af81c6b444171045cc614fd1f409358ebc810d55f053c8dcde9'],
+    }),
+    ('Class::Inspector', '1.32', {
+        'source_tmpl': 'Class-Inspector-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE'],
+        'checksums': ['cefadc8b5338e43e570bc43f583e7c98d535c17b196bcf9084bb41d561cc0535'],
+    }),
+    ('Parse::RecDescent', '1.967015', {
+        'source_tmpl': 'Parse-RecDescent-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JT/JTBRAUN'],
+        'checksums': ['1943336a4cb54f1788a733f0827c0c55db4310d5eae15e542639c9dd85656e37'],
+    }),
+    ('Carp', '1.50', {
+        'source_tmpl': 'Carp-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['f5273b4e1a6d51b22996c48cb3a3cbc72fd456c4038f5c20b127e2d4bcbcebd9'],
+    }),
+    ('XML::Parser', '2.44', {
```

Comment:
@Micket Unintended downgrade? Latest is `2.44_01`, like we have in existing `Perl` easyconfigs, see also `XML-Parser-*.tar.gz` at https://cpan.metacpan.org/authors/id/T/TO/TODDR/

Reply:
That was indeed unintentional. I blame metacpan:
https://fastapi.metacpan.org/v1/release/XML-Parser
Doesn't show 2.44_01. :-1:

Reply:
Must be the funky versioning...